src/index.mako.html

```diff
@@ -214,6 +210,9 @@
         <div ga-geolocation ga-geolocation-map="map"></div>
   % endif
         <div id="zoomButtons"></div>
+  % if device == 'mobile' or device == 'embed':
+        <div ga-background-selector ga-background-selector-map="map" class="ga-bg-mobile"></div>
```

Comment:
Adding a css class here suggest the css  is defined on the app.css, but it is not. I think this class should be added dynamically in the directive. The same for `ga-bg-desktop`.